tests/performance/timing_step_37.cc

```diff
@@ -537,7 +503,7 @@ template <int dim>
 void
 LaplaceProblem<dim>::setup_transfer()
 {
-  mg_transfer.setup(mg_matrices, mg_constrained_dofs);
+  mg_transfer.initialize_constraints(mg_constrained_dofs);
```

Comment:
@kronbichler Was there a particular reason to use `MGTransferMF` in the performance test?

Reply:
I think it is copy-paste, in order to get the code work without `adjust_ghost_range_if_necessary` as in the basic `MGTransferMatrixFree`. I think it is not really necessary due to the stored partitioners anyway, but I did not check.

Reply:
Does the performance change?